frontend/src/__tests__/cypress/cypress/pages/servingRuntimes.ts

```diff
@@ -40,31 +40,35 @@ class ServingRuntimes {
     this.wait();
   }
 
+  findAppTitle() {
+    return cy.findByTestId('app-page-title');
+  }
+
   private wait() {
     this.findAddButton();
     cy.testA11y();
   }
 
   shouldBeMultiModel(enabled = true) {
-    cy.findByText('Multi-model serving enabled').should(enabled ? 'exist' : 'not.exist');
+    cy.findByTestId('multi-model-serving-enabled').should(enabled ? 'exist' : 'not.exist');
     return this;
   }
 
   shouldBeSingleModel(enabled = true) {
-    cy.findByText('Single-model serving enabled').should(enabled ? 'exist' : 'not.exist');
+    cy.findByTestId('single-model-serving-enabled').should(enabled ? 'exist' : 'not.exist');
     return this;
   }
 
   findAddButton() {
-    return cy.findByRole('button', { name: 'Add serving runtime' });
+    return cy.findByTestId('add-serving-runtime-button');
   }
 
   findStartFromScratchButton() {
     return cy.findByRole('button', { name: 'Start from scratch' });
```

Comment:
please add data-test id here

Reply:
These selectors  are not defined in the code (its inside the <codeEditor>, so I don't find a way to add data-testid to them.
@christianvogt are we aiming at changing this as well?

Reply:
Not much we can do here since it's part of the PF code editor. Leave as is.